packages/dev/core/src/Physics/v2/physicsBody.ts

```diff
@@ -173,6 +188,28 @@ export class PhysicsBody {
         return this._physicsPlugin.getEventMask(this);
     }
 
+    public setMotionType(motionType: PhysicsMotionType): void {
+        this._physicsPlugin.setMotionType(this, motionType);
+    }
+
+    public getMotionType(): PhysicsMotionType {
+        return this._physicsPlugin.getMotionType(this);
+    }
+
+    /**
+     * Computes the mass properties of the physics object, based on the set of physics shapes this body uses.
+     *
+     * @param massProps - The mass properties to set.

```

Comment:
There is no param :-)

Reply:
this code has been discussed in this PR. see previous comments.

Reply:
Fixed